hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -371,6 +371,8 @@
                    to send or received a byte will depend on !!COMMAND.SPEED.
```

Comment:
THis is not part of this CL, but why does this have a `!!` in it? That syntax is confusing to some readers, and should be `foo != 0`, instead

Reply:
Apparently, it's confusing to reviewers too :-) This is a magic syntax that generates a link to the register in the generated documentation.

Reply:
Yeah I believe we had also used this syntax in the markdown file before introducing Hugo shortcodes.

Reply:
Oh, goodness.  @msfschaffner and @rswarbrick have it.    I was unaware that this had been changed.  Is there a good example for how this is supposed to look now?

Reply:
Inside the Hjson you still have to use the `!!` prefix. 
In the markdown, we use the Hugo shortcode `{{< regref "REGISTER_NAME" >}}`.

Reply:
Oh got it.  So nothing has changed then.  Thanks @msfschaffner